.github/workflows/conda-build.yml

```diff
@@ -8,14 +8,14 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v4  # Update to latest version
 
     - name: Set up Miniconda
-      uses: conda-incubator/setup-miniconda@v2
+      uses: conda-incubator/setup-miniconda@v3
       with:
         auto-update-conda: true
-        python-version: 3.8
-        channels: conda-forge,defaults,bioconda
+        python-version: 3.11
+        channels: conda-forge,bioconda,defaults
 
     - name: Create conda environment
       run: conda env create -f environment.yml
```

Comment:
Why are you creating a Conda environment with ALL dependencies before starting the build? This is usually not how it works.
Conda build will install those requirements for you based on the meta.yml in the recipe.

Reply:
It might not fail but it should be unnecessary

Reply:
Got it.